R/plots.R

```diff
@@ -158,6 +159,17 @@ get_param_label <- function(x, id_val) {
   res
 }
 
+paste_param_by <- function(x) {
+  if (".by" %in% colnames(x)) {
+    x <- x %>% dplyr::mutate(.metric = case_when(
+      !is.na(.by) ~ paste0(.metric, "(", .by, ")"),
```

Comment:
Yup, heard. This one seemed like a wash in terms of readability as it takes some understanding of the glue package to realize that `()` are intended to be kept rather than part of the syntax.